src/harness/testcases/WINNF_FT_S_FDB_testcase.py

```diff
@@ -949,7 +966,9 @@ def generate_FDB_6_default_config(self, filename):
         'grantRequests': [
             [grant_g1_a, grant_g1_b],
             [grant_g2_a, grant_g2_b]],
-        'expectedResponseCodes': [(400,), (0,)],
+        'expectedResponseCodes': [
+            [(400,), (0, 400)],
```

Comment:
The check should be for 0 OR 400, not both. Please check the spec.

Reply:
Updated   as per the comment